src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3342,7 +3342,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <ItemGroup>
       <IntermediateSatelliteAssembliesWithTargetPath Include="$(IntermediateOutputPath)%(EmbeddedResource.Culture)\$(TargetName).resources.dll"
-                                                     Condition="'%(EmbeddedResource.Culture)' != ''">
+                                                     Condition="'%(EmbeddedResource.Culture)' != '' and '$(MSBuildRuntimeType)' != 'Core'">
```

Comment:
Yeah, since it should keep the list empty so the existing condition on the `Copy` task should skip it.